symphony/lib/toolkit/class.widget.php

```diff
@@ -710,7 +710,7 @@ public static function Select($name, array $options = null, array $attributes =
      */
     private static function __SelectBuildOption($option)
     {
-        @list($value, $selected, $desc, $class, $id, $attributes) = $option;
+        list($value, $selected, $desc, $class, $id, $attributes) = array_pad($option, 6, null);
```

Comment:
My guess was that the silencer was introduced because the array may be incomplete. But meanwhile this is not enough. (I may be wrong, as I say, it was just a guess.)

You know me. I prefer loud noise when things break — this means that they can be fixed.